docs/ReleaseNotes/ReleaseNotes_Android.md

```diff
@@ -1,4 +1,8 @@
 # MobileID SDK - Release Notes
+## 7.1.0
+- Added improved security measures
```

Comment:
How about changing to "Improved security measures" 
Maybe specify some details regarding hash and hybrid encryption